### PR TITLE
Add glTF icon

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
image
I don't know why the M in the ™ is cut off in this preview. I promise it's not like this in the svg.

**Popularity metric:**
https://www.similarweb.com/website/khronos.org/#overview
https://github.com/KhronosGroup/glTF
https://trends.google.com/trends/explore?date=all&q=%2Fg%2F11c2j8dkh8&hl=en
"glTF is the 'JPEG of 3D'" - [This page](https://www.khronos.org/gltf/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
I have made sure that everything is to the standards of both Khronos® Group and Simple Icons.

### CODE REVIEW

**Comment by anhthang** [comment]
You SVG is not optimized with [SVGO](https://svgomg.net/) and the view box is `0 0 25 25`. That's why the M is cut off in the preview

**Comment by MichaelHinrichs** [comment]
> Your SVG is not optimized with [SVGO](https://svgomg.net/)

Yes it is. Did I get a setting wrong?

**Comment by uncenter** [comment]
> > Your SVG is not optimized with [SVGO](https://svgomg.net/)
> 
> Yes it is. Did I get a setting wrong?

It's mostly fine, I think the only outstanding issue is you have an extra `version="1.1"` attribute that isn't needed. Please address the viewbox issue as well as @anhthang pointed out.

**Comment by MichaelHinrichs** [comment]
Oh, also just now seeing that i got the viewbox wrong. But wouldn't making it smaller cut off more of the logo?

**Comment by uncenter** [comment]
> Oh, also just now seeing that i got the viewbox wrong. But wouldn't making it smaller cut off more of the logo?

You can edit the viewbox in the text file from 25x25 to 24x24, open it in your editor, and resize the icon to fit in the proper 24x24 viewbox. Nothing will be lost.

**Comment by MichaelHinrichs** [comment]
New preview.
image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6133,6 +6133,12 @@
             "hex": "F2CC38",
             "source": "https://about.glovoapp.com/press"
         },
+        {
+            "title": "glTF",
+            "hex": "87c540",
```
Comment (uncenter) [inline _data/simple-icons.json:6138]:
```suggestion
            "hex": "86C540",
```
This is what I'm seeing from their logo assets zip.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (MichaelHinrichs) [reply _data/simple-icons.json:6138]: https://www.khronos.org/files/legal/Khronos-Logo-Family-Color-Guide.pdf

Comment (uncenter) [reply _data/simple-icons.json:6138]: Ah looks like the logo usage PDF uses `#87C540?`

Comment (uncenter) [reply _data/simple-icons.json:6138]: We'll just stick with the logo usage guide / what you got already.

**Comment by uncenter** [comment]
See the lint errors please.

**Comment by MichaelHinrichs** [comment]
I'm not sure I understand. Do I need to capitalize the c in the hex code?

**Comment by uncenter** [comment]
> I'm not sure I understand. Do I need to capitalize the c in the hex code?

Yes. See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#7-update-the-json-data-for-simpleiconsorg.

File: icons/glTF.svg
Comment (uncenter) [inline icons/glTF.svg:1]: The filename here should be all lowercase.

Comment (MichaelHinrichs) [reply icons/glTF.svg:1]: Oh, ok.

**Comment by uncenter** [comment]
Please see the linter errors. Your precision level is too high, try optimizing with 4 or 5.

File: icons/gltf.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...02-.2989.36896-.51017.49399-.21126.12531-.46718.1881-.76804.1881-.30085 0-.55147-.0625-.75185-.1881-.20067-.12503-.3617-.28997-.48338-.49399-.12196-.20401-.20764-.43118-.25788-.68208-.0502-.25062-.0751-.50124-.0751-.75186 0-.26485.0304-.5191.0913-.76274.0608-.24336.1574-.45826.28997-.64441.1323-.18615.30253-.33463.51017-.4457.20764-.11108.45826-.16662.75186-.16662.28634 0 .5297.0575.73035.17192.2004.11471.36337.26681.48868.45659.12531.18977.21658.40272.2738.6391.0572.23639.086.4764.086.71977.00027.25759-.0301.51547-.091.77334zm.79819 4.21308c-.52608.47752-1.46604 1.02286-2.7462 1.23691.85708.14904 1.7602.22858 2.69289.22885 3.6069.00084 6.76558-1.25839 8.53665-3.04343-1.61813 1.06639-4.042 1.67312-6.74632 1.67312-.64218-.00028-1.13364-.0229-1.73702-.0954zm5.5535-2.08142h-1.31058V9.7579h-1.64436V8.63514h4.5996V9.7579h-1.64438v4.41542Zm3.63732 0h-1.31086V8.63514H22.021V9.7579h-2.73559v1.03625h2.39455v1.12276h-2.39455Zm-8.2224-6.91798h.91289v6.90207h-.91289zm12.0908 1.54223h-.20514v.53333h-.17916v-.53333h-.20513v-.15406h.58914zm.0801-.15406h.26373l.11833.45826h.002l.11833-.45826H24v.68739h-.16857v-.52189h-.002l-.14429.52189h-.13675l-.14457-.52189h-.002v.52189h-.16849v-.68739z"/></svg>
```
Comment (uncenter) [inline icons/gltf.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.67095 15.73118c.41193-.46189.62962-1.137.62962-2.04626V8.60444h-.86992v.79484h-.0106c-.16494-.30784-.39379-.53864-.68739-.69269-.2936-.15378-.61593-.23081-.96674-.23081-.47975 0-.88443.0913-1.21375.27378-.32932.18253-.59444.41724-.79483.70358-.20066.28634-.34383.60143-.4298.94526-.086.34384-.12893.68042-.12893 1.00974 0 .37955.0519.73958.15573 1.0795.10382.34021.25955.63911.46719.89698.20763.25788.46551.46189.77334.61232.30783.15043.66953.2255 1.08509.2255.17192 0 .3455-.0215.52105-.0645.17555-.043.3402-.10549.49425-.1881.15378-.0823.29165-.18615.41361-.31146.12168-.12531.22187-.27379.30085-.4457h.0212v.36504c0 .31509-.0304.60143-.0913.85931-.0608.25787-.15935.47807-.29527.66059-.13619.18253-.3081.3257-.51547.4298-.20763.10382-.46188.15573-.76245.15573-.15043 0-.30812-.0162-.4725-.0483a1.88243 1.88243 0 0 1-.17331-.0427 10.19466 10.19466 0 0 1-.18168-.0659c-.006-.002-.0112-.005-.0168-.007v.001c-2.35966-.88163-3.91221-2.37949-3.91221-4.0794 0-2.71132 3.94961-4.90912 8.82187-4.90912 2.7141 0 5.14772.61817 6.76613 1.69098-1.7666-1.79759-4.93366-3.06074-8.55506-3.06129C4.49689 5.16042.00056 7.97091 0 11.43912c-.00028 2.3672 2.09453 4.42881 5.18819 5.49966 2.48134.0849 3.74085-.37565 4.48266-1.2076Zm-.37454-3.68952c-.0608.25788-.15573.48868-.28466.69269-.12894.20402-.2989.36896-.51017.49399-.21126.12531-.46718.1881-.76804.1881-.30085 0-.55147-.0625-.75185-.1881-.20067-.12503-.3617-.28997-.48338-.49399-.12196-.20401-.20764-.43118-.25788-.68208-.0502-.25062-.0751-.50124-.0751-.75186 0-.26485.0304-.5191.0913-.76274.0608-.24336.1574-.45826.28997-.64441.1323-.18615.30253-.33463.51017-.4457.20764-.11108.45826-.16662.75186-.16662.28634 0 .5297.0575.73035.17192.2004.11471.36337.26681.48868.45659.12531.18977.21658.40272.2738.6391.0572.23639.086.4764.086.71977.00027.25759-.0301.51547-.091.77334zm.79819 4.21308c-.52608.47752-1.46604 1.02286-2.7462 1.23691.85708.14904 1.7602.22858 2.69289.22885 3.6069.00084 6.76558-1.25839 8.53665-3.04343-1.61813 1.06639-4.042 1.67312-6.74632 1.67312-.64218-.00028-1.13364-.0229-1.73702-.0954zm5.5535-2.08142h-1.31058V9.7579h-1.64436V8.63514h4.5996V9.7579h-1.64438v4.41542Zm3.63732 0h-1.31086V8.63514H22.021V9.7579h-2.73559v1.03625h2.39455v1.12276h-2.39455Zm-8.2224-6.91798h.91289v6.90207h-.91289zm12.0908 1.54223h-.20514v.53333h-.17916v-.53333h-.20513v-.15406h.58914zm.0801-.15406h.26373l.11833.45826h.002l.11833-.45826H24v.68739h-.16857v-.52189h-.002l-.14429.52189h-.13675l-.14457-.52189h-.002v.52189h-.16849v-.68739z"/></svg>
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
See lint errors. Make sure the title in the SVG is properly capitalized (*not* kebab-lower-cased like the filename), and the SVG needs to be centered.

**Comment by MichaelHinrichs** [comment]
I thought it was centered.

**Comment by uncenter** [comment]
The linter is complaining again about unoptimized/too high precision on the path elment, and there cannot be a trailing newline on the SVG file.

**Comment by uncenter** [comment]
Great work, thanks!

**Pull request merged**